### ISSUE TITLE
terminate process

### ISSUE THREAD

**Issue by mwilliams31** [fireeye/capa-rules#299]
## Summary

`terminate process` false positive that's hitting on the `__invoke_watson` function.

## Examples

Sample SHA256: `563dd5a95f439bc2b4170a74c8be565a1af076e6cbebd1d018b2809a1e8bc908`
Offset: `0x39DC`

## Possible improvements

Rule out non-library hits based on the `TerminateProcess` `uExitCode` value (i.e., `0xC0000417`)?

**Comment by williballenthin** [comment]
we may be able to address this via FLIRT, too

**Comment by mr-tz** [comment]
Thanks for the report, that's what we need to make the tool even better!
+1 on FLIRT, otherwise, we can use the constant per your suggestion!

**Comment by mr-tz** [comment]
FLIRT recognizes `DEBUG:viv_utils.flirt:found library function: 0x100045dc: __invoke_watson`.

However, there's another FP on an interesting case where `0x10005e1d` is recognized as `___report_gsfailure`, but

```
terminate process (2 matches)
...
function @ 0x10004C96
  or:
    and:
      or:
        api: kernel32.TerminateProcess @ 0x10005F1B
```

still hits.

`0x10004C96` is `__security_check_cookie` which jumps to `___report_gsfailure`.


https://github.com/mandiant/capa/issues/989 should help here.
